code/modules/events/event_container.dm

```diff
@@ -194,7 +194,7 @@ var/list/event_last_fired = list()
 		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Meteor Wave",		/datum/event/meteor_wave,		0,						list(ASSIGNMENT_ENGINEER =  5),	1),
 		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Abductor Visit",	/datum/event/abductor, 		    80, is_one_shot = 1),
 		new /datum/event_meta/alien(EVENT_LEVEL_MAJOR, "Alien Infestation",	/datum/event/alien_infestation, 		0,		list(ASSIGNMENT_SECURITY = 15), 1),
-		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Traders",			/datum/event/traders,			0, is_one_shot = 1),
+		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Traders",			/datum/event/traders,			180, is_one_shot = 1),
```

Comment:
This value was based on a conversation I had with Neca. The idea is that at first, we set it high (base chance equal to blob on reasonably highpop), and then, after a week or two, we review it and adjust its chances based on the impact it has on rounds. During this period, it will regularly inform crew that they've missed the chance of a trade ship due to remaining on red alert a lot, which may be a substantial discouragement from remaining on red all the time. It will also slightly reduce the chances of other major events. The idea has been brought up that it should re-roll a new, different major event if it doesn't happen due to red alert... but the idea has also been brought up that major events currently have too big an impact on the round, and this reduces that somewhat, so its a good thing. I await more head/maint feedback before I consider implementing this "reroll if trader event happens on red" idea, or otherwise tweaking it.